app/web/features/search/SearchBox.tsx

```diff
@@ -55,42 +56,47 @@ export default function SearchBox({
   const { t } = useTranslation([GLOBAL, SEARCH]);
   const classes = useStyles();
 
-  const { control, setValue, errors, register, handleSubmit } = useForm({
+  const { control, errors } = useForm({
     mode: "onChange",
   });
 
-  function handleOnChangeAutocomplete(event: "" | GeocodeResult) {
+  const handleOnChangeAutocomplete = (event: GeocodeResult) => {
```

Comment:
why const rather normal function? (just to understand)

Reply:
It's not a huge deal either way, const seems a bit more modern to me.

Technically, using const creates a function expression which has a few advantages:

1. block-scoped behavior - meaning the function is only available within the block it's defined in, which gives better control over where the function can be accessed.
2. prevents re-declaration - using const guarantees the function can't be accidentally re-declared or reassigned within the same block, which adds a layer of safety.

Both are valid though!

Reply:
📝📝📝 good info... will keep it!!